### ISSUE TITLE
THIS ISNT AN ISSUE !

### ISSUE THREAD

**Issue by Leejit** [SalmonGER/StatsPE#41]
Hello, Amazing Plugin !! 

I'm looking for compatibility with Tesseract. 

Thank You.

**Comment by SalmonDE** [comment]
How is it incompatible?

**Comment by Aericio** [comment]
It works perfectly for me. (factions server is running off of Tesseract)

**Comment by Leejit** [comment]
LOL!! Close this.... This is why I shouldn't work on my server when I'm dogg ass tired at 4:30am. 

I meant to post this on TopVoter xD

**Comment by inxomnyaa** [comment]
\*facepalms the hell out of myself\*

**Comment by Leejit** [comment]
.....lol. why ?

**Comment by Leejit** [comment]
Also, just wondering... can someone blurr their api key, but show me how
they setup their config... I'm seriously struggling with it... and it's
probably just something stupid I'm not doing right.

Sorry guys... I seem to be getting dumber in my older age.

**Comment by SalmonDE** [comment]
@LordJoshie This isn't the TopVoter repo but you're probably doing it right just with the wrong API key. Check if it's correct.

**Comment by Leejit** [comment]
I know, I moved my issue to the proper location. the issue is closed,
because it isnt an issue wirh Salmons work, so it's just a discussion to
help me solve my issue. I have checked the key with the website URL and see
the info stored for my voters, but the plugin still isnt receiving the data
correctly.

Ive ran this exact plugin and config for it with PocketMine, and it worked.
I'm sure it will work on Tess also, I'm just doing something wrong.

Thanks tho !
